apps/web_app/telegram/handlers/create_notification.py

```diff
@@ -0,0 +1,132 @@
+from aiogram import F, Router, types
+from aiogram.fsm.context import FSMContext
+from aiogram.fsm.state import State, StatesGroup
+from database.models import NotificationData
+from telegram.crud import TelegramCrud
+from utils.values import ProtocolIDs
+from fastapi import status
+
+create_notification_router = Router()
+
+
+class NotificationFormStates(StatesGroup):
+    """States for the notification form process."""
+    wallet_id = State()
+    health_ratio_level = State()
+    protocol_id = State()
+
+
+@create_notification_router.callback_query(F.data == "create_subscription")
+async def start_form(callback: types.CallbackQuery, state: FSMContext):
+    """Initiates the notification creation form by asking for the wallet ID."""
+    await state.set_state(NotificationFormStates.wallet_id)
+    await callback.message.edit_text(
+        "Please enter your wallet ID:",
+        reply_markup=types.InlineKeyboardMarkup(
+            inline_keyboard=[
+                [types.InlineKeyboardButton(text="Cancel", callback_data="cancel_form")]
+            ]
+        ),
+    )
+
+
+@create_notification_router.message(NotificationFormStates.wallet_id)
+async def process_wallet_id(message: types.Message, state: FSMContext):
+    """Processes the wallet ID input from the user."""
+    await state.update_data(wallet_id=message.text)
+    await state.set_state(NotificationFormStates.health_ratio_level)
+    await message.answer(
+        "Please enter your health ratio level (between 0 and 10):",
+        reply_markup=types.InlineKeyboardMarkup(
+            inline_keyboard=[
+                [types.InlineKeyboardButton(text="Cancel", callback_data="cancel_form")]
+            ]
+        ),
+    )
+
+
+@create_notification_router.message(NotificationFormStates.health_ratio_level)
+async def process_health_ratio(message: types.Message, state: FSMContext):
+    """Processes the health ratio level input from the user."""
+    try:
```

Comment:
use try except only to expected lines which can failed `float(message.text)` can be just covered with try/except other code can be moved out